src/components/NavBar/Styles.ts

```diff
@@ -62,6 +62,9 @@ const useStyles = makeStyles((theme) => ({
         fontSize: '15px',
         width: '100%',
     },
+    textWhite: {
+        color: 'white',
```

Comment:
use theme color, even if it is white
theme.palette.common.white

Reply:
okay will do